Makefile

```diff
@@ -275,13 +275,13 @@ clean: clean-cross-build
 .PHONY: clean
 
 licensecheck: microshift bin/lichen
-	bin/lichen -c .lichen.yaml microshift
+	$(GO_BUILD_BINDIR)/lichen -c .lichen.yaml $(GO_BUILD_BINDIR)/microshift
```

Comment:
We have a different pattern for installing the tools now. It goes via the scrtips/fetch_tools.sh script.
Can we update this code to use this script, which will also make the lichen exe location more standard under _output/bin, etc.?

Reply:
Sure, moved over the installation to the script